test/io/inputs.jl

```diff
@@ -58,3 +58,94 @@ end
 
     @test match == true
 end
+
+@testset "ZeroDataCube" begin
+    ZeroDataCube = ADRIA.ZeroDataCube
+    
+    dim_1_name = :timesteps
+    dim_1_vals = rand(10)
+    dim_1 = Dim{dim_1_name}(dim_1_vals)
+    
+    dim_2_name = :location
+    dim_2_vals = ["loc 1", "loc 2", "loc 3", "loc 4", "loc 5"]
+    dim_2 = Dim{dim_2_name}(dim_2_vals)
+
+    yax_res = ZeroDataCube(Int, timesteps=dim_1_vals)
+
+    @test typeof(yax_res) <: YAXArray{Int, 1} || 
+        "Incorrect return type. Expected a subtype of YAXArray{Int, 1} \
+         but received $(typeof(yax_res))"
+
+    @test all(0 .== yax_res) || 
+        "Incorrect data contained in YAXArray. Expected all 0."
+
+    @test size(yax_res) == (10,) ||
+        "Incorrect YAXArray shape. Expected (10,) but received $(size(yax_res))"
+
+    @test name(dim_1) == name(yax_res.axes[1]) ||
```

Comment:
You don't need to build a `Dim` just for this.  `name(yax_res.axes[1]) == dim_1_name` does the same thing and is more economic. 

Also, you could have used this variable `dim_1_name` to build the `ZeroDataCube` - this will make it easier to update this test in the future, if needed:

```
dim_1_name = (:timesteps,)
dim_1_vals = ((rand(10)),)
ADRIA.ZeroDataCube(Int; NamedTuple{dim_1_name}(dim_1_vals)...)
```